R/folders.R

```diff
@@ -1,3 +1,12 @@
+#' Get metadata cache subfolder name
+#'
+#' Change subfolder name in the code of this function for metadata cache here to apply globally, as all functions in the package should use this function to get the metadata cache path.
+#' @return Character string with the subfolder name for the raw data cache.
+#' @keywords internal
+spod_subfolder_metadata_cache <- function() {
```

Comment:
Do we we really need a function that just returns a single text string? Not sure how `spod_subfolder_metadata_cache()` could be better than "metadata_cache" but imagine there's a reason..

Reply:
maybe it is a bit of over engineering, but at least we will only have to change it in one place if we want to change it. better solutions? package option?

Reply:
No better solutions, just checking the logic and fine by me for this to go into the pkg.